tensorflow/python/training/experimental/loss_scale.py

```diff
@@ -197,6 +197,10 @@ def from_config(cls, config):
     """Creates the LossScale from its config."""
     return cls(**config)
 
+  def variables(self):
```

Comment:
I would remove this change, and just access `loss_scale._weights` in loss_scale_optimizer.py. Use the `# pylint: disable=protected-access` comment to disable the lint error. The reason is that I am unsure if we want to expose this function as a public API.

Reply:
Removed